modulemd/tests/ModulemdTests/translation_helpers.py

```diff
@@ -137,6 +138,36 @@ def test_translations_from_catalog(self):
                                 profile.get_description(
                                     str(catalog.locale)), msg_string)
 
+    @mock.patch('ModulemdTranslationHelpers.Utils.KojiSessionMock')
+    def test_index_from_tags(self, mock_session):
+        tags = ['f29']
+
+        # Create a Modulemd.ModuleIndex object containing the YAML stream
+        idx = Modulemd.ModuleIndex.new()
+        self.assertIsNotNone(idx)
+        ret, failures = idx.update_from_file(
+            "%s/f29.yaml" %
+            (os.getenv('TEST_DATA_PATH')), True)
+        self.assertTrue(ret)
+        self.assertTrue(len(failures) == 0)
+
+         # Test when koji returns proper modules
+        index = Utils.get_index_from_tags(mock_session, tags)
```

Comment:
Before calling into `Utils.get_index_from_tags()`, you need to set up the mock functions for `mock_session`. For example:

```python3
        koji_session_mock = Utils.KojiSessionMock()

        mock_session.listTagged.side_effect = koji_session_mock.listTagged
        mock_session.getBuild.side_effect = koji_session_mock.getBuild
```